### PR TITLE
Like operator for fetch

### PR DESCRIPTION
**Pull request by ShrijanL**
- Added like operator for fetch function.
- Added like testcase.

### CODE REVIEW

File: django_generic_api/payload_models.py [hunk 1]
```diff
@@ -23,6 +23,7 @@ class OperatorByEnum(str, Enum):
     IN = "in"
     NOT = "not"
     GT = "gt"
+    LIKE = "like"
```
Comment (nareshelequin) [inline django_generic_api/payload_models.py:26]: extend with `ilike` too

File: django_generic_api/tests/test_fetch.py [hunk 2]
```diff
@@ -134,6 +134,43 @@ def test_fetch_filter_operator_in(
...
+        self, fetch_data_1, api_client, view_perm_token
+    ):
+        fetch_payload = {
+            "payload": {
+                "variables": {
+                    "modelName": "customer",
+                    "fields": ["name", "email"],
+                    "filters": [
+                        {
+                            "operator": "like",
```
Comment (nareshelequin) [inline django_generic_api/tests/test_fetch.py:147]: what if user does `Like` , `lIke`, `liKE`?

File: django_generic_api/tests/test_fetch.py [hunk 3]
```diff
@@ -134,6 +134,43 @@ def test_fetch_filter_operator_in(
...
+        headers = {"Authorization": f"Bearer {view_perm_token}"}
+
+        response = api_client.post(
+            "/fetch/",
+            fetch_payload,
+            format="json",
+            headers=headers,
+        )
+        response_data = json.loads(response.content.decode("utf-8"))
+        assert response.status_code == 200
```
Comment (nareshelequin) [inline django_generic_api/tests/test_fetch.py:168]: do assert on address, bcz is really doing job as required for `address like 'hyd'`?

File: django_generic_api/tests/test_fetch.py [hunk 4]
```diff
@@ -134,6 +134,43 @@ def test_fetch_filter_operator_in(
             {"name": "test_user1", "email": "user1@gmail.com"},
         ]
 
+    def test_fetch_filter_operator_like(
```
Comment (nareshelequin) [inline django_generic_api/tests/test_fetch.py:137]: -ve test case?

**Review by nareshelequin — CHANGES_REQUESTED** [review]

File: django_generic_api/tests/test_fetch.py (hunk 3 above)
Comment (ShrijanL) [reply django_generic_api/tests/test_fetch.py:300]: Added address in assert for like operator

File: django_generic_api/payload_models.py (hunk 1 above)
Comment (ShrijanL) [reply django_generic_api/payload_models.py:26]: added 'ilike' operation

**Pull request merged**